spec/api-app-spec.js

```diff
@@ -238,7 +239,7 @@ describe('app module', () => {
     let server = null
     const socketPath = process.platform === 'win32' ? '\\\\.\\pipe\\electron-app-relaunch' : '/tmp/electron-app-relaunch'
 
-    beforeEach((done) => {
+    beforeEach(done => {
```

Comment:
Do we have it somewhere in the style guide?
That brackets should be omitted if there is only one argument.

Reply:
in some cases yes, but since omitting brackets adds an implicit `return` i tend not to do that lest i introduce side effects

Reply:
I meant just `done =>` instead of `(done) =>`.
There's no difference expect of a code style.

Reply:
on oh sorry, brackets typically mean `[` and `]` so i got confused :)